mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/clients/vedtak/VeilarbvedtaksstotteClientImpl.kt

```diff
@@ -41,31 +41,35 @@ class VeilarbvedtaksstotteClientImpl(
     }
 
     override suspend fun hentSiste14AVedtak(fnr: String, accessToken: String): VedtakDto? {
-        return CacheUtils.tryCacheFirstNotNull(siste14aVedtakCache, fnr) {
-            val response = client.get("$baseUrl/siste-14a-vedtak?fnr=$fnr") {
-                bearerAuth(tokenProvider.invoke(accessToken))
-            }
-
-            if (response.status == HttpStatusCode.NotFound || response.status == HttpStatusCode.NoContent) {
-                log.info("Fant ikke siste 14A-vedtak for bruker")
-                return null
-            }
+        return CacheUtils.tryCacheFirstNullable(siste14aVedtakCache, fnr) {
+            try {
+                val response = client.get("$baseUrl/siste-14a-vedtak?fnr=$fnr") {
+                    bearerAuth(tokenProvider.invoke(accessToken))
+                }
 
-            val body = response.bodyAsText()
-            if (body.isBlank()) {
-                log.info("Fant ikke siste 14A-vedtak for bruker")
-                return null
-            }
+                if (response.status == HttpStatusCode.NotFound) {
+                    log.info("Fant ikke siste 14A-vedtak for bruker")
+                    return@tryCacheFirstNullable null
+                } else if (response.status == HttpStatusCode.Forbidden) {
```

Comment:
Siden det ikke har blitt sjekket for eksplisitte responskoder her så har vi forsøkt å parse alle responser som dto's og falt tilbake til å returnere `null`. 

Egentlig tror jeg denne oppførselen blir litt feil. Noen av disse veilarbtjenestene kan gi andre `403`-responser enn det vi allerede sjekker for i vår tjeneste, f.eks. dette kallet til `siste-14a-vedtak`. Denne responsen blir maskert i form av at vi bare returnerer `null` og så blir det vist en melding til veileder i frontenf, f.eks: 
```Brukeren har ikke fått §14 a-vedtak enda, og kan derfor ikke meldes på noen tiltak.```

Det som _egentlig_ kan skje er at veilederen mangler en tilgang og da burde oppførselen hos oss vært `403 Forbidden` heller enn fallback til at brukeren ikke har fått 14a-vedtak (for det kan jo hende den faktisk har det): https://github.com/navikt/veilarbvedtaksstotte/blob/21e52256239cd13531a85a136bbeca8d48532b18/src/main/java/no/nav/veilarbvedtaksstotte/controller/Siste14aVedtakController.kt#L31

Jeg har ikke gjort noen endringer i oppførsel her enda fordi jeg ikke er helt sikker på hvordan vi burde gjøre dette i frontend, kanskje noe vi kan se på sammen?